common/src/command.rs

```diff
@@ -1,7 +1,85 @@
 use enum_dispatch::enum_dispatch;
+use ssip::Priority;
 
 use strum::{Display, EnumDiscriminants};
 
+pub trait IntoCommands {
+	fn into_commands(self) -> Vec<OdiliaCommand>;
```

Comment:
We need to capture self since this is an `into_` method, not a `to_` method. We'd need to actually take ownership of strings to pass them across channels.

It should absolutely be a capture. Even if we decide against using channels, the capture makes more sense so we don't have to deal with too much ownership BS.